build.sbt

```diff
@@ -8,23 +8,23 @@ homepage := Some(url("http://github.com/cardamo/play-logback-access"))
 
 licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-version := "0.8.0-SNAPSHOT"
+version := "0.9.0-SNAPSHOT"
 
 libraryDependencies ++= Seq(
   "org.playframework" %% "play" % "3.0.1",
-  "ch.qos.logback" % "logback-access" % "1.4.7",
+  "ch.qos.logback" % "logback-access" % "1.4.14",
   "jakarta.servlet" % "jakarta.servlet-api" % "6.0.0" % Optional,
   //https://github.com/playframework/playframework/releases/tag/2.8.15
-  "com.google.inject"            % "guice"                % "5.1.0",
-  "com.google.inject.extensions" % "guice-assistedinject" % "5.1.0"
+  "com.google.inject"            % "guice"                % "6.0.0",
+  "com.google.inject.extensions" % "guice-assistedinject" % "6.0.0"
 )
 
 libraryDependencies ++= Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % "7.0.1" % Test,
-  "org.playframework" %% "play-pekko-http-server" % "3.0.1" % Test
+  "org.playframework" %% "play-pekko-http-server" % "3.0.2" % Test
 )
 
-scalaVersion := "2.13.5"
```

Comment:
Removing this seem to enable compilation for unsupported 2.12.
A also get compilation errors for 2.13, for example:
```
[error] /home/asinicyn/ws/play-logback-access/src/main/scala/module.scala:68:94: not found: type ?
[error]   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[?]] = Seq(
```

Reply:
hi @cardamo , thanks
I see, it was tested with the cross-compiling command "sbt ++compile" "sbt ++test". It worked fine.
I set the default the default version to 2.13.13 now. It would be nice to push both artifacts to the repo.